docs/ur/docs/benchmarks.md

```diff
@@ -1,4 +1,4 @@
-# بینچ مارکس
+# <div style="text-align: right;">بینچ مارکس</div>
```

Comment:
Is this necessary? 🤔 

I would think maybe there's then a configuration missing in MkDocs... I'll have to investigate it.

Reply:
@tiangolo previously it was left aligned so I have to add this you can check and let me know. Also I simply inherit the parent (en) MkDocs in my ur folder MkDocs file.